python/tvm/relay/op/vision/nms.py

```diff
@@ -0,0 +1,36 @@
+"""Non-maximum suppression operations."""
+from __future__ import absolute_import as _abs
+from . import _make
+
+def nms(data,
+        valid_count,
+        nms_threshold=0.5,
+        force_suppress=False,
+        nms_topk=-1):
+    """Generate prior(anchor) boxes from data, sizes and ratios.
+
+    Parameters
+    ----------
+    data : relay.Expr
+        3-D tensor with shape [batch_size, num_anchors, 6].
+        The last dimension should be in format of
+        [class_id, score, box_left, box_top, box_right, box_bottom].
+
+    valid_count : relay.Expr
```

Comment:
why is valid_count used here?

Reply:
The methods multibox_detection and box_nms generating valid_cout are slightly different. This NMS is a general interface to support both cases. I put valid_count as an argument so that we don't need to add extra flag to differentiate multibox_detection and box_nms use cases.